dags/taar_daily.py

```diff
@@ -0,0 +1,275 @@
+        job_name="TAAR_similarity",
+        python_driver_code="gs://moz-fx-data-prod-airflow-dataproc-artifacts/jobs/taar_similarity.py",
+        num_workers=4,
+        worker_machine_type="n1-highmem-96",
+        master_machine_type='n1-standard-8',
+        py_args=[
+            "--date", "{{ ds_nodash }}",
+            "--bucket", "telemetry-private-analysis-2",
+            "--prefix", "taar/similarity/",
+            "--aws_access_key_id", taar_aws_access_key,
+            "--aws_secret_access_key", taar_aws_secret_key,
+        ],
+        aws_conn_id=taar_aws_conn_id,
+        gcp_conn_id=taar_gcpdataproc_conn_id,
+    ),
+    dag=dag,
+)
+
+taar_collaborative_recommender = SubDagOperator(
+    task_id="addon_recommender",
+    subdag=moz_dataproc_jar_runner(
+        parent_dag_name=dag.dag_id,
+        dag_name="addon_recommender",
+        job_name="Train_the_Collaborative_Addon_Recommender",
+        main_class="com.mozilla.telemetry.ml.AddonRecommender",
+        jar_urls=[
+            "https://s3-us-west-2.amazonaws.com/net-mozaws-data-us-west-2-ops-ci-artifacts"
+            "/mozilla/telemetry-batch-view/master/telemetry-batch-view.jar",
+        ],
+        jar_args=[
+          "train",
+          "--runDate={{ds_nodash}}",
+          "--inputTable=gs://moz-fx-data-derived-datasets-parquet/clients_daily/v6",
+          "--privateBucket=s3a://telemetry-parquet",
+          "--publicBucket=s3a://telemetry-public-analysis-2",
+        ],
+        cluster_name="addon-recommender-{{ds_nodash}}",
+        image_version="1.3",
+        worker_machine_type="n1-standard-8",
+        num_workers=20,
+        optional_components=[],
+        install_component_gateway=False,
+        init_actions_uris=[],
+        aws_conn_id=taar_aws_conn_id,
+        gcp_conn_id=taar_gcpdataproc_conn_id,
+        default_args={
+            key: value
+            for key, value in chain(default_args.items(), [
+                ("owner", "mlopatka@mozilla.com"),
+                ("email", ["telemetry-alerts@mozilla.com", "mlopatka@mozilla.com", "vng@mozilla.com"]),
+            ])
+        },
+    ),
+    dag=dag,
+)
+
+taar_lite = SubDagOperator(
+    task_id="taar_lite",
+    subdag=moz_dataproc_pyspark_runner(
+        parent_dag_name=dag.dag_id,
+        dag_name="taar_lite",
+        default_args=default_args,
+        cluster_name=taarlite_cluster_name,
+        job_name="TAAR_Lite_GUID_GUID",
+        python_driver_code="gs://moz-fx-data-prod-airflow-dataproc-artifacts/jobs/taar_lite_guidguid.py",
+        num_workers=8,
+        py_args=[
+            "--date",
+            "{{ ds_nodash }}",
+            "--aws_access_key_id",
+            taar_aws_access_key,
+            "--aws_secret_access_key",
+            taar_aws_secret_key,
+        ],
+        aws_conn_id=taar_aws_conn_id,
+        gcp_conn_id=taar_gcpdataproc_conn_id,
+    ),
+    dag=dag,
+)
+
+
+amodump >> amowhitelist
+amodump >> editorial_whitelist
```

Comment:
Minor nit - editorial_whitelist doesn't need to be downstream from amodump.  The editorial whitelist is generated directly off of an AMO REST API.